content/swiftui/concepts/viewmodifier/terms/padding/padding.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.padding()'
+Description: 'Applies padding to the view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.padding()`** modifier method applies padding to the view.
+
+## Syntax
+
+```psuedo
+Text("This text has padding around it.")
+    .padding(edge, number)
+```
+
+The `.padding()` modifier method has the following attributes:
+
+- `edge` to specify the edge on which the padding should be applied. One or multiple edges or a collection of edges can be specified. For example, `.top` will add padding only to the view's top edge, `[.bottom, trailing]` will add padding to the bottom and right edge of the view, and `.vertical`  will add padding to the left and right edge of the view. If no edge is specified, the padding will be applied on all edges of the view.
```

Comment:
I'd like to see a bulleted list of all the `edge` options.

Reply:
My suggestion was to include *all* the possible options. For instance, there's no mention of `.horizontal` or how to add padding to the left edge.

Reply:
Thank you, @SSwiniarski, for the clarification! Wouldn't a table be a better option than a sub-list?

Reply:
@cslylla, either way would work. Whatever you're comfortable with.

Reply:
@SSwiniarski, I tried the table, but it made the document more difficult to read. So I listed all the options. I am curious to see what you think!

Reply:
@cslylla The bulleted list works well.